<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>577</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>677</width>
    <height>577</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>677</width>
    <height>577</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>YouTube Video Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/youtube_icon.png</normaloff>:/icons/icons/youtube_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#pushButton{
	background-color: rgb(212, 212, 212);
	border-radius:5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Source</string>
    </property>
    <widget class="QLineEdit" name="sourceLineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#sourceLineEdit{
	border-radius:10px;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Enter YouTube Video URL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="convertButton">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>15</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:pushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;

}</string>
     </property>
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Destination</string>
    </property>
    <widget class="QLineEdit" name="destinationLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#destinationLineEdit{
	border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Enter Destination Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:pushButton{
	color:rgba(255, 255, 255, 210);
	border-radius:5px;

}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="videoGruopBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>631</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Video</string>
    </property>
    <widget class="QPushButton" name="downloadButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#downloadButton
{
	background-color: rgb(28, 173, 73);
	border-radius:5px;
	color:	rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QComboBox" name="video_Quality_ComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox#video_Quality_ComboBox{
}</string>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="videoImage">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>120</width>
       <height>66</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="video_tittle">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>VÄ°DEO ADI</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>55</y>
       <width>291</width>
       <height>10</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar#progressBar
{
	background-color: rgb(218, 218, 218);
	color: rgb(9, 180, 66);
	border-radius:1px
}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QLabel" name="videoTime">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#videoTime
{
color: rgb(83, 83, 83);
}</string>
     </property>
     <property name="text">
      <string>5:35</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
